source/test/main.test.ts

```diff
@@ -24,7 +24,7 @@ import {
 
 import {
   MainStack,
-} from '../src/molecular-unfolding/cdk/stack-main';
+} from '../src/cdk/stack-main';
```

Comment:
add test cases for covering architect changing

Reply:
the changes should include,

- SNS resource
- email subscription
- IAM role changes of Notebook
- etc.